chart/templates/_helpers.tpl

```diff
@@ -68,6 +75,22 @@ add below linux tolerations to workloads could be scheduled to those linux nodes
     - windows
 {{- end -}}
 
+
+{{- define "required-scheduling-node-selector-terms" -}}
+requiredDuringSchedulingIgnoredDuringExecution:
+           nodeSelectorTerms: {{ include "linux-node-selector-terms" . | nindent 14 }}
```

Comment:
Instead of applying indentation in a helper function, you should apply the indentation after the helper function renders the YAML. This would avoid weird spacing stuff like this.